pkg/kapis/cluster/v1alpha1/handler.go

```diff
@@ -362,14 +375,171 @@ func validateKubeSphereAPIServer(ksEndpoint string, kubeconfig []byte) (*version
 	response.Body = ioutil.NopCloser(bytes.NewBuffer(responseBytes))
 
 	if response.StatusCode != http.StatusOK {
-		return nil, fmt.Errorf("invalid response: %s , please make sure ks-apiserver.kubesphere-system.svc of member cluster is up and running", responseBody)
+		return nil, fmt.Errorf("invalid response: %s , please make sure %s.%s.svc of member cluster is up and running", KubeSphereApiServer, KubesphereNamespace, responseBody)
 	}
 
 	ver := version.Info{}
 	err = json.NewDecoder(response.Body).Decode(&ver)
 	if err != nil {
-		return nil, fmt.Errorf("invalid response: %s , please make sure ks-apiserver.kubesphere-system.svc of member cluster is up and running", responseBody)
+		return nil, fmt.Errorf("invalid response: %s , please make sure %s.%s.svc of member cluster is up and running", KubeSphereApiServer, KubesphereNamespace, responseBody)
 	}
 
 	return &ver, nil
 }
+
+// validateMemberClusterConfiguration compares host and member cluster jwt, if they are not same, it changes member
+// cluster jwt to host's, then restart member cluster ks-apiserver.
+func (h *handler) validateMemberClusterConfiguration(memberKubeconfig []byte) error {
+	hConfig, err := h.getHostClusterConfig()
+	if err != nil {
+		return err
+	}
+
+	mConfig, err := h.getMemberClusterConfig(memberKubeconfig)
+	if err != nil {
+		return err
+	}
+
+	if hConfig.AuthenticationOptions.JwtSecret != mConfig.AuthenticationOptions.JwtSecret {
+		return fmt.Errorf("hostcluster Jwt is not equal to member cluster jwt, please edit the member cluster cluster config")
```

Comment:
how about `JWT token of member cluster is not the same with host cluster, please double check configmap kubesphere-config under namespace kubesphere-system`

Reply:
What does `double check` mean? Do we need to check the existence of the kubesphere-config configmap?

Reply:
means "check again"

Reply:
Why do we need to check again? We have already checked the configMap between the host and member cluster. We can not compare the cc cause there doesn't exist the cc informers. The installer team just writes a crd and uses the shell operator to watch cc reources.

Reply:
That is just a widely used `phrase`, which means users have to make sure jwtToken from configmap be the same with the host cluster's no matter how.